docs/intro/migrations.rst

```diff
@@ -21,69 +39,77 @@ your codebase.
-  │       └── 00001.edgeql
-  │       └── ...
+  │   └── default.esdl          # schema file (written by you)
   └── edgedb.toml
 
-The schema itself is written using EdgeDB's schema definition language.
+The schema itself is written using EdgeDB's schema definition language. Edit
+your ``dbschema/default.esdl`` and add the following schema inside your
+``module default`` block:
 
 .. code-block:: sdl
 
   type User {
-    required property name -> str;
+    required name: str;
   }
 
   type Post {
-    required property title -> str;
-    required link author -> User;
+    required title: str;
+    required author: User;
   }
 
 
-It's common to keep your entire schema in a single file, typically called
-``default.esdl``. However it's also possible to split it across a number of
-``.esdl`` files.
-
-To spin up a new instance and populate it with an initial schema, execute the
-following commands in a fresh directory.
+It's common to keep your entire schema in a single file, and many users use
+this ``default.esdl`` that is created when you start a project. However it's
+also possible to split their schemas across a number of ``.esdl`` files.
 
-.. code-block:: bash
+Once you save your initial schema, assuming it is valid, the ``watch`` command
+will pick it up and apply it to your database.
 
-  $ edgedb project init
-  Do you want to initialize a new project? [Y/n]
-  > Y
-  <additional prompts>
-  $ edgedb migration create
-  Created dbschema/migrations/00001.edgeql (id: <hash>)
-  $ edgedb migrate
-  Applied dbschema/migrations/00001.edgeql (id: <hash>)
 
 2. Edit your schema files
 -------------------------
 
 As your application evolves, directly edit your schema files to reflect your
-desired data model.
+desired data model. Try updating your ``dbschema/default.esdl`` to add a
+``Comment`` type:
 
 .. code-block:: sdl-diff
 
     type User {
-      required property name -> str;
+      required name: str;
     }
 
-    type BlogPost {
-      property title -> str;
-      required link author -> User;
+    type Post {
+      required title: str;
+      required author: User;
     }
 
   + type Comment {
-  +   required property content -> str;
+  +   required content: str;
   + }
 
+When you save your changes, the updates will be immediately applied to your
+database as long as ``watch`` is still running.
+
+.. note::
+
+    If your schema cannot be applied, the ``watch`` command will generate an
+    error. If you're using one of our client bindings as you update your schema
+    with ``watch``, you will see the error there the next time you execute a
+    query using that client binding.
+
+    If things aren't working the way you expect after making a schema change,
+    take a look at the ``watch`` console to find out why.
+
+Once you have the schema the way you want it, and you're ready to lock it in
+and commit it to version control, stop your ``watch`` command with Ctrl-c. It's
```

Comment:
You don't have to stop ``watch`` to run ``migration create`` or commit. Did you see some failures if it doesn't?

Reply:
No, this was a bad assumption on my part.